angrmanagement/ui/main_window.py

```diff
@@ -565,6 +565,7 @@ def _save_database(self, file_path):
         angrdb.dump(file_path)
 
         self.workspace.instance.database_path = file_path
+        self.workspace.plugins.handle_project_save(file_path)
```

Comment:
This callback should be put before the database is serialized, in case users want to make any changes to the data to be saved.